test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/EnvironmentTagHelperTest.cs

```diff
@@ -66,6 +66,97 @@ public void ShowsContentWhenCurrentEnvironmentIsNotSet(string namesAttribute, st
             ShouldShowContent(namesAttribute, environmentName);
         }
 
+
+        [Theory]
+        [InlineData("", "", "", "Development")]
+        [InlineData("", "", "Test", "Development")]
+        [InlineData("Development", "", "", "Development")]
+        [InlineData("", "Development", "", "Development")]
+        [InlineData("", "Development", "Test", "Development")]
+        [InlineData("Development", "", "Test", "Development")]
+        [InlineData("Development", "Development", "", "Development")]
+        [InlineData("Test", "Development", "", "Development")]
+        [InlineData("Development", "Test", "", "Development")]
+        [InlineData("Development", "Development", "Test", "Development")]
+        [InlineData("Development", "Test", "Test", "Development")]
+        [InlineData("Test", "Development", "Test", "Development")]
```

Comment:
Test `null` as well as empty strings. Same for the data added below. (Note: You don't really need to test every positive `""`, `"Development"`, `"Test"` combination.)

Also suggest removing the `environmentName` parameter since its value never varies.